.bazelrc

```diff
@@ -266,6 +266,8 @@ common --experimental_profile_include_target_label
 common --experimental_profile_include_primary_output
 # Don't merge timing profile actions.
 common --noslim_profile
+# Include compact execution log
+common --experimental_execution_log_compact_file=compact_exec.log
```

Comment:
It would be nice if we could have a flag to always write this file to the output base by default and upload it to BES after (similar to how timing profile works)

Reply:
added as TODO